met-web/src/components/scrollToTop/index.tsx

```diff
@@ -6,6 +6,10 @@ const ScrollToTop = () => {
 
     useEffect(() => {
```

Comment:
I like this, can we please have it in its own separate component though. We can name it something relevant and specific to Formio. FormioModalListener... maybe

Reply:
For sure ill do that now